packages/tradershub/src/features/cfd/modals/MT5PasswordModal/MT5ChangePassword.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { Fragment, useEffect, useState } from 'react';
+import { passwordRegexDescription } from '@/constants';
+import { useMT5AccountHandler, useQueryParams } from '@/hooks';
+import { validPassword } from '@/utils';
+import { useTradingPlatformPasswordChange } from '@deriv/api-v2';
+import { Button, Modal, PasswordInput, Text } from '@deriv-com/ui';
+import { CFDPlatforms } from '../../constants';
+
+const MT5ChangePassword = () => {
+    const { isLoading: tradingPlatformPasswordChangeLoading, mutateAsync: tradingPasswordChange } =
+        useTradingPlatformPasswordChange();
+    const { handleSubmit: createMT5Account, createMT5AccountStatus } = useMT5AccountHandler();
+    const { openModal } = useQueryParams();
+
+    const [currentPassword, setCurrentPassword] = useState('');
+    const [newPassword, setNewPassword] = useState('');
+
+    const changePasswordAndCreateAccountHandler = async () => {
+        await tradingPasswordChange({
+            new_password: newPassword,
+            old_password: currentPassword,
+            platform: CFDPlatforms.MT5,
+        }).then(() => createMT5Account(newPassword));
```

Comment:
```suggestion
        await tradingPasswordChange({
            new_password: newPassword,
            old_password: currentPassword,
            platform: CFDPlatforms.MT5,
        })
         createMT5Account(newPassword);
```

Not an ideal thing to combine `async/await` & callback chaining